tailwind.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     logs: true,
     rtl: false,
     prefix: '',
-    darkTheme: 'dark',
+    darkTheme: 'light',
```

Comment:
On running `npm run dev` the application was in dark mode. This was different to the loom so I thought to change it to light mode. This definitely isn't the way to do it but just for quickness added it here